helpers/validator/validator_test.go

```diff
@@ -0,0 +1,111 @@
+package validator
+
+import (
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+)
+
+type testPair struct {
+	value  string
+	result bool
+}
+
+var testsEmail = []testPair{
+	{randString(91) + "@" + randString(4) + "." + randString(3), true},
+	{randString(50) + "@gmail.com", true},
+	{randString(10) + "@" + randString(5) + "." + randString(3), true},
+	{randString(20), false},
+	{randString(99), false},
+	{randString(50) + "@" + randString(10), false},
+	{randString(10) + "." + randString(4), false},
+	{randString(100), false},
+}
+
+var testsMessage = []testPair{
+	{randString(3999), true},
+	{randString(int(src.Int63() % 400)), true},
+	{strings.Repeat(" ", 50) + randString(1), true},
+	{strings.Repeat(" ", 50) + randString(3950), true},
+	{randString(4001), false},
+	{strings.Repeat(" ", 100), false},
+}
+
+var testsName = []testPair{
+	{randString(29), true},
+	{randString(1), true},
+	{randString(1) + strings.Repeat(" ", 29), true},
+	{randString(1) + strings.Repeat(" ", 30), false},
+	{randString(31), false},
+	{strings.Repeat(" ", int(src.Int63()%31)), false},
+}
+
+func TestIsValidEmail(t *testing.T) {
+	for _, pair := range testsEmail {
```

Comment:
If you rename _testsEmail_ to _emailTests_, you could rename _pair_ to _emailTest_ (same below).